### DISCUSSION TITLE
how Lighthouse timespan score works

### DISCUSSION THREAD

**Discussion by swatigautam011** [GoogleChrome/lighthouse#16119] [Q&A] [answered]
I am using timespan mode for my application, and the score is 16/19 and is in amber, where all the metrics (TBT, CLS, and IP) are green. Ideally, when all the metrics are green, my score should also be green. Here, we are not sure what to fix to make the score look good. And please explain how the score is defined for timespan mode; "16/19" could mean that out of 19, 16 metrics met the goals.

**Comment by adamraine** [comment, marked as answer]
It means 16/19 audits passed (not just metrics). This method of summarizing timespan performance mode isn't ideal for sure, future changes to the UX are tracked in #14559
